### DISCUSSION TITLE
Strange MinGW build error, my fault?

### DISCUSSION THREAD

**Discussion by ell1e** [nothings/stb#1244] [Q&A] [answered]
On latest HEAD I get this:

```
vendor/stb/stb_vorbis.c:5129:27: error: expected identifier or '(' before numeric constant
 5129 | #define PLAYBACK_RIGHT    4
      |                           ^
vendor/stb/stb_vorbis.c:5128:27: error: expected identifier or '(' before numeric constant
 5128 | #define PLAYBACK_LEFT     2
      |                           ^
vendor/stb/stb_vorbis.c:5128:27: error: expected identifier or '(' before numeric constant
 5128 | #define PLAYBACK_LEFT     2
```

But only in MinGW targeting Windows, the Linux build is fine. This is how I included it:

```
#include <stb/stb_vorbis.c>
```

Anyone got a clue if that's a bug or if I did something wrong? Any ideas would be appreciated, I'm a bit stumped.

**Comment by nothings** [comment]
putting stb_vorbis.c in a #include is not supported, if you're going to do it you're on your own!

**Reply by ell1e** [reply]
Oh I see! But I'm confused now, how else am I supposed to use it? There is no accompanying header file, is there?

**Comment by nothings** [comment, marked as answer]
You can either cut and paste the "header-file section" into your own stb_vorbis.h, or you can #define STB_VORBIS_HEADER_ONLY before you #include it (every time you include it).
